e2e/preview/pipelines/test_rag_pipelines.py

```diff
@@ -12,10 +12,8 @@
 from haystack.preview.components.builders.prompt_builder import PromptBuilder
 
 
-@pytest.mark.skipif(
-    not os.environ.get("OPENAI_API_KEY", None),
-    reason="Export an env var called OPENAI_API_KEY containing the OpenAI API key to run this test.",
-)
+# TODO enable again once we have stabilized generators components and formats
```

Comment:
This does not seem related to this PR.
Has anything major changed in GPTGenerator?

Reply:
Yes, good catch. I'll reenable these tests